migrations/Version20221208235505.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Delete this migration file. the creation of the job table is done in the previous migration file. When running migrations will get an error that the table jobs already exists